content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -25,6 +25,8 @@ For more information on the listed features, visit the [Okta Glossary](https://h
 6. Visit the "Sign On" tab in the application.
 7. Copy the "Identity Provider Metadata" URL.
 
+For information on configuring automated Team Mapping using Okta Group Membership, please see the [Configuration - Team Mapping (Okta)](./okta.html.md#configuration---team-mapping-okta) section below. 
```

Comment:
The term "team mapping" should not be capitalized. 

I strongly suspect that "Okta group membership" shouldn't be capitalized either, but I don't know for sure because I'm not as familiar with Okta. Is the exact phrase "Okta Group Membership" used within Okta as consistent trade dress? Or no?

When linking _within_ a page, please omit the path and filename of the page -- you only need the part after the `#` character. Also, you've included the `.md` file extension after the `.html` extension, so this link won't even work -- the .md extension is removed during the site build process. (Note that every other link on the site points to a .html file rather than a .html.md file.)

Reply:
Thanks.  Updated.